README.md

```diff
@@ -53,11 +53,13 @@ This provider's versions are able to install and manage the following versions o
 
 This provider's versions are able to install Kubernetes to the following versions of OpenStack:
 
-| | OpenStack Pike | OpenStack Queens | OpenStack Rocky | OpenStack Stein | OpenStack Train |
-|-------------------------------------------------|---|---|---|---|---|
-| OpenStack Provider v1alpha1 (release-0.1 branch)| ✓ | ✓ | ✓ | ✓ | + |
-| OpenStack Provider v1alpha2 (v0.2)              | + | ✓ | + | ✓ | + |
-| OpenStack Provider v1alpha3 (v0.3)              | + | + | + | ✓ | ✓ |
+| | OpenStack Pike | OpenStack Queens | OpenStack Rocky | OpenStack Stein | OpenStack Train | OpenStack Ussuri | OpenStack Victoria |
+|-------------------------------------------------|---|---|---|---|---|---|---|
+| OpenStack Provider v1alpha1 (release-0.1 branch)| ✓ | ✓ | ✓ | ✓ | + | + | + |
+| OpenStack Provider v1alpha2 (v0.2)              | + | ✓ | + | ✓ | + | + | + |
+| OpenStack Provider v1alpha3 (v0.3)              | + | + | + | ✓ | ✓ | ✓ | ✓ |
+
+**NOTE:** : There is no formal CI test for specific OpenStack version, the development team has a CI running on latest OpenStack devstack and that covers most of `✓` at above table.
```

Comment:
This is not true. Our OpenStack version in CI is still Ussuri.
https://github.com/kubernetes-sigs/cluster-api-provider-openstack/blob/master/.zuul/playbooks/run.yaml#L27

Reply:
CI tests only non HA version. Some OpenStack versions for HA are tested manually.

Reply:
understand .. thanks for point out
so we need some manual test but actually CI is preferred

let me check how we can do and submit some proposal for review

Reply:
Ideally using CAPI e2e test framework is the best to test HA. I tried but still [in progress](https://github.com/hidekazuna/cluster-api-provider-openstack/tree/e2e). I confirmed it tests HA.

So in this PR, I propose to remove Victoria, and update NOTE:
The development team has a CI using [non HA template](./templates/cluster-template-without-lb.yaml) running on latest OpenStack devstack and that covers most of `✓` at above table. Some versions are tested against HA.